src/IR.cpp

```diff
@@ -907,6 +907,46 @@ Stmt Atomic::make(const std::string &producer_name,
     return node;
 }
 
+namespace {
+
+const char *const instruction_op_names[] = {
+    // Shared:
+    "abs",
+    "dot_product",
+    "rounding_halving_add",
+    "saturating_add",
```

Comment:
Summary from offline discussion: We disagree whether or not this overlap with Call intrinsics should exist, but are tabling it for now.